src/components/organization/organization.service.ts

```diff
@@ -61,6 +62,25 @@ export class OrganizationService {
       canReadName: true,
     };
     try {
+      const checkOrg = await this.db
+        .query()
+        .raw(
+          `
+          MATCH(org:OrgName {value: $name}) return org
+          `,
+          {
+            name: input.name,
+          }
+        )
+        .first();
+
+      if (checkOrg) {
+        throw new BadRequestException(
+          'Organization with that name already exists.',
+          'Duplicate'
+        );
+      }
```

Comment:
Actually this will get caught by the catch below and change the error type. So it needs to be moved out of the try/catch

Reply:
Ah, correct.